feat: added more questions to absolute and relative units quiz (#58441)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/quiz-css-relative-and-absolute-units/66ed9009f45ce3ece4053eb7.md

#### --text--

> Which unit is the commonest absolute unit in CSS?

Reviewer: ```suggestion
Which unit is the most commonly used absolute unit in CSS?
```

#### --text--

Which unit is relative to the font size of the root element?

#### --distractors--

> Pixels

Reviewer: ```suggestion
pixels
```

---

> Point

Reviewer: ```suggestion
point
```

---

> percentage

Reviewer: we will probably want the wrong answers to be in backticks too.
otherwise, only the correct answer will be in backticks making it super obvious what to pick

Author: Should be fine now
